packages/core/types/Fields.ts

```diff
@@ -94,7 +94,10 @@ export type ExternalFieldWithAdaptor<
   placeholder?: string;
   adaptor: Adaptor<any, any, Props>;
   adaptorParams?: object;
-  getItemSummary: (item: NotUndefined<Props>, index?: number) => string;
+  getItemSummary: (
+    item: NotUndefined<Props>,
+    index?: number
+  ) => string | ReactNode;
```

Comment:
```suggestion
  ) => ReactNode;
```
ReactNode includes strings, numbers, or anything that can be used as React children.